os/drivers/input/ist415.c

```diff
@@ -503,14 +523,18 @@ static void ist415_stop_device(struct ist415_dev_s *dev)
 		ist415_disable(dev);
 		ist415_power_off(dev);
 	}
+
+	return OK;
```

Comment:
Could you let me know why you change the return type even this returns OK always?

Reply:
This is ops in touchscreen, so some touch ic might have return fail